thunder/torch/__init__.py

```diff
@@ -1761,6 +1761,18 @@ def real(a):
 # TODO Move these to torch.nn.functional
 
 
+@torchsymbol(torch.celu, torch.nn.functional.celu, id="torch.celu", is_method=True)
+def celu(a: TensorLike, /, alpha: float = 1.0, inplace: bool = False) -> TensorLike:
+    negative_domain_value = expm1(a / alpha) * alpha
+    out = where(a > 0, a, negative_domain_value)
+    if inplace:
+        return prims.copy_(out, a)
+    return out
+
+
+_inplace_to_out_of_place[celu] = celu, 1
```

Comment:
@crcrpar, why is the index of the `inplace` argument needed in the code? What happens if it's specified as a keyword argument and not passed positionally? Are there tests that should fail if the index is wrong?

Hopefully Masaki can explain what's going on here but it seems from the signature the index should be 2.
```suggestion
_inplace_to_out_of_place[celu] = celu, 2
```